packages/settings/src/lib/photon.ts

```diff
@@ -218,7 +219,8 @@ const organizationSettings: {
     ...defaultSettings,
     logo: 'blueberry_logo.png',
     accentColor: '#235AFF',
-    enableMedHistory: true
+    enableMedHistory: true,
+    topRankedWalgreens: true
```

Comment:
you don't want to set this now right? you'll have a PR for this on Monday?

Reply:
hmm didn't mean to make this a "review"

Reply:
removed